addons/calendar/tests/test_event_recurrence.py

```diff
@@ -376,6 +378,36 @@ def test_all_day_date(self):
         self.assertEqual(events[0].start_date, date(2019, 10, 22), "The first event has the initial start date")
         self.assertEqual(events[1].start_date, date(2019, 10, 29), "The start date of the second event is one week later")
 
+    def test_recurrency_with_this_event(self):
+        """
+        1) Create an event with a recurrence set on it
+        2) Try updating the event with a different recurrence without specifying 'recurrence_update'
+        3) Update the recurrence of one of the events, this time using the 'recurrence_update' as future_events
+        4) Finally, check that the updated event correctly reflects the recurrence
+        """
+        event = self.env['calendar.event'].create({
+            'name': "Test Event",
+            'allday': False,
+            'rrule': u'FREQ=DAILY;INTERVAL=1;COUNT=10',
+            'recurrency': True,
+            'start': fields.Datetime.to_string(datetime.now() + timedelta(days=10)),
+            'stop': fields.Datetime.to_string(datetime.now() + timedelta(days=15)),
```

Comment:
Can you use freezetime or just set a hard-coded date instead of using the current day?

Reply:
Hello, @leonardopavanrocha 
Sure,  changes are applied. :slightly_smiling_face: